docs/prfc/drafts/pimp.xml

```diff
@@ -268,27 +260,25 @@
      </section>
      
      <section title = "Error handling">
-       <t>There are two types of errors that can occur:</t>
+       <t>There are multiple errors that can occur:</t>
        
        <t><list style="numbers">
-				 <t>Packet loss</t>
-         <t>Corrupted data (indicated by an incorrect checksum)</t>
+		        <t>Packet loss</t>
+            <t>Corrupted data (indicated by an incorrect checksum)</t>
+		        <t>*Packet with invalid flags (invalid flag combination) is received</t>
        </list></t>
        
-       <t>Both of these cases are handled the same way. A packet
+       <t>The first cases are handled the same way. A packet
      requesting retransmission of the lost or corrupted packet
      will be sent to the caller.</t>
+     	<t>*In the last case (invalid flag combination for a packet received), such as an incorrect combination of flags, the packet will be dropped.</t>
+
      </section>
    </section>
    
    <section title="Technical Functionality">
-     <t>PIMP retains the standard TCP three way handshake and
-     termination procedures. Additionally, when the reciever
-     sends the FIN packet, they specify the final sequence number
-     they have sent. This gives the server one last chance to ask
-     for any retranmits in the (shortened, final) window, and wait
-     until they are recieved before sending a FIN packet back to the 
-     reciever."</t>
+     <t>PIMP retains the standard TCP three way handshake for connection establishment and
+     termination procedures. The handshake includes the following steps: 1) SYN: Client sends a SYN packet (with SYN flag) to the server, seqNum set to random uint32 'A'. 2) SYN-ACK: Server responds with a SYN-ACK packet (such that SYN and ACK flags are set), this packet sets the ackNum (unit32) to the received seqNum A + 1. the seqNum is set to a new random uint32 'B'. 3) ACK: Client acknowledges by sending an ACK packet (with ACK flag set) to the server, where seqNum is set to received ackNum, and the ackNum is set to the received seqNum + 1. If this is completed, connections is established as both sides have acknowledged the connetion. Additionally, when the server sends the FIN packet, they specify the final sequence number they have sent. This gives the server one last chance to ask for any retranmits in the (shortened, final) window, and wait until they are received before sending a FIN packet back to the reciever."</t>

```

Comment:
The three-way handshakes make sense since it's just like TCP.

Reply:
We already have the PRFC updated for handshake. What does this add?

Also, please do not reference TCP. Write this from scratch.